house_rules/Item_Rules/boots_of_speed.md

```diff
@@ -0,0 +1,9 @@
+# Boots of Speed
+
+* [Original Item Description](https://www.dndbeyond.com/magic-items/4589-boots-of-speed)
+
+## Modifications
```

Comment:
Please don't put changelog timestamps on the page. git tracks this for us and it doesn't benefit the end consumer.

Reply:
:+1: